src/components/organisms/Chatbox/Chatbox.tsx

```diff
@@ -46,24 +52,25 @@ const Chatbox: React.FunctionComponent<PropsType> = ({
   );
 
   const { user } = useUser();
-  const chats = useSelector((state) => state.firestore.ordered.venueChats);
-  const privateChats = useSelector(
-    (state) => state.firestore.ordered.privatechats
-  );
   const users = useSelector(partygoersSelectorData);
   const userArray = useSelector(partygoersSelector) ?? [];
 
-  useFirestoreConnect({
-    collection: "privatechats",
-    doc: user?.uid,
-    subcollections: [{ collection: "chats" }],
-    storeAs: "privatechats",
-  });
-
   const [searchValue, setSearchValue] = useState<string>("");
   const debouncedSearch = debounce((v) => setSearchValue(v), 500);
   const searchRef = useRef<HTMLInputElement>(null);
 
+  useFirestoreConnect(
+    user && user.uid
+      ? {
+          collection: "privatechats",
+          doc: user.uid,
+          subcollections: [{ collection: "chats" }],
+          storeAs: "privatechats",
+        }
+      : undefined
+  );
```

Comment:
Here's the pathological query

Reply:
Hope it resolves it. Seemed ok in my inspection.